tools/config.ts

```diff
@@ -72,8 +72,10 @@ export const DEV_NPM_DEPENDENCIES: InjectableDependency[] = normalizeDependencie
 ]);
 
 export const PROD_NPM_DEPENDENCIES: InjectableDependency[] = normalizeDependencies([
+  { src: 'systemjs/dist/system-polyfills.src.js', inject: 'shims' },
```

Comment:
That one is required, I'm getting an error otherwise `Uncaught ReferenceError: $__curScript is not defined` ... the app works but not the change detection.